tests/core/mempool/test_mempool_manager.py

```diff
@@ -37,7 +37,7 @@
 TEST_COIN3 = Coin(IDENTITY_PUZZLE_HASH, IDENTITY_PUZZLE_HASH, TEST_COIN_AMOUNT3)
 TEST_COIN_ID3 = TEST_COIN3.name()
 TEST_COIN_RECORD3 = CoinRecord(TEST_COIN3, uint32(0), uint32(0), False, TEST_TIMESTAMP)
-TEST_HEIGHT = uint32(1)
+TEST_HEIGHT = uint32(5)
```

Comment:
I think this constant is a mistake. apparently it wasn't used for anything until now